src/Servers/HttpSys/src/RequestProcessing/Request.cs

```diff
@@ -48,9 +48,9 @@ internal Request(RequestContext requestContext)
             SslStatus = requestContext.SslStatus;
 
             KnownMethod = requestContext.VerbId;
-            Method = requestContext.GetVerb();
+            Method = requestContext.GetVerb()!;
 
-            RawUrl = requestContext.GetRawUrl();
+            RawUrl = requestContext.GetRawUrl()!;
```

Comment:
When does GetVerb() and GetRawUrl() return null?

Reply:
https://github.com/dotnet/aspnetcore/blob/f46385410ac61017dbc06cde1d21e3f99aad8e2d/src/Shared/HttpSys/RequestProcessing/NativeRequestContext.cs#L171-L178

Reply:
Do we ever actually expect to call GetRawUrl() and get null back? Can it just throw instead?

Reply:
Hmm, I think that is a decision for the caller of GetRawUrl. In the case of this code in `Request`, to get to this point we know that we're processing a HTTP request so raw url and friends will always have a value.